app/models/provider.rb

```diff
@@ -8,9 +8,6 @@ class Provider < ApplicationRecord
 
   validates :name, presence: true
   validates :dttp_id, uniqueness: true, format: { with: /\A[a-f0-9]{8}-([a-f0-9]{4}-){3}[a-f0-9]{12}\z/i }
-  validates :code, format: { with: /\A[A-Z0-9]+\z/i }
```

Comment:
Wondering if we need to keep in the validation regardless and allow blank values?

Reply:
right added that back in, and allow blanks